### PR TITLE
Interfaces for nested types

### PR DESCRIPTION
**Pull request by dmdashenkov**
This PR fixes generating interfaces for nested Protobuf messages.

Before this PR, the `(is)` and `(every_is)` options were ignored for nested messages. Starting from now, they are treated the same for top-level and nested messages.

Important notice: `EventMessage`, `CommandMessage`, and `RejectionMessage` interfaces are not applied to nested messages declared in the corresponding files.
However, `UuidValue` interface may be applied to both top-level and nested messages.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: base/src/test/proto/spine/test/code/proto/test_commands.proto [hunk 1]
```diff
@@ -17,21 +17,27 @@
...
- */
-@Immutable
-final class EventMessageClassifier extends MessageClassifier {
+option (type_url_prefix) = "type.spine.io";
+option (SPI_all) = true;
+option java_package = "io.spine.test.code.proto.command";
+option java_multiple_files = true;
+option java_outer_classname = "MttCommandsProto";
+
+// This file define test environment for `MessageTypeTest.java`.
```
Comment (armiol) [inline base/src/test/proto/spine/test/code/proto/test_commands.proto]:
```suggestion
// This file defines test environment for `MessageTypeTest.java`.
```

File: base/src/test/proto/spine/test/code/proto/test_events.proto [hunk 2]
```diff
@@ -17,21 +17,27 @@
...
- */
-@Immutable
-final class CommandMessageClassifier extends MessageClassifier {
+option (type_url_prefix) = "type.spine.io";
+option (SPI_all) = true;
+option java_package = "io.spine.test.code.proto.event";
+option java_multiple_files = true;
+option java_outer_classname = "MttEventsProto";
+
+// This file define test environment for `MessageTypeTest.java`.
```
Comment (armiol) [inline base/src/test/proto/spine/test/code/proto/test_events.proto]: Same as above.

File: base/src/test/proto/spine/test/code/proto/test_events.proto [hunk 3]
```diff
@@ -17,21 +17,27 @@
...
+
+// A sample event to verify telling that a message is an event from `MessageType`.
```
Comment (armiol) [inline base/src/test/proto/spine/test/code/proto/test_events.proto]:
```suggestion
// A sample event to verify detecting that a message is an event from `MessageType`.
```

**Pull request merged**